roles/sysstat/tasks/common.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Install packages
+  apt:
+    pkg: "{{ item }}"
+    state: latest
+    force: yes
+  with_items:
```

Comment:
A lot of these should be in `common` probably - not sure what all goes where so wanted to ask.

Not sure why we need `libncurses` for a server